structuredefinitions/Extension-UKCore-GenomeSequencingSampleTypeCategory.xml

```diff
@@ -52,7 +52,7 @@
       <binding>
         <strength value="preferred" />
         <description value="A set of codes to record the state of the sample" />
```

Comment:
This, and the extension description/purpose, probably want updating to say "the whole genome sequencing type of the sample" rather than state?

Reply:
Though I think Omar is going to provide some text as well